fiona/errors.py

```diff
@@ -1,6 +1,10 @@
 # Errors.
 
 
+class FionaError(Exception):
```

Comment:
Could it cause some confusion that not all exceptions raised by Fiona inherit from this? e.g. `DriverError` is not a `FionaError` (because it's a `FionaValueError`).